.github/workflows/python-package-ci.yml

```diff
@@ -15,12 +15,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ 3.6, 3.7, 3.8, 3.9 ] # FIXME github recognizes 3.10 as 3.1 lol
+        python-version: [ 3.7, 3.8, 3.9, '3.10', 3.11-dev ]
```

Comment:
Could you add back python3.6 to this list? I think some of our users still rely on this version and we want to have them covered :)